src/components/Garden/Filters/ListFilter.js

```diff
@@ -26,19 +28,22 @@ function ListItem({ index, item, onSelect, selected }) {
 
   return (
     <div
-      onClick={handleOnClick}
       css={`
         margin-bottom: ${1 * GU}px;
-        color: ${theme.contentSecondary};
-
-        ${selected === index
-          ? `
-          color:${theme.content}
-        `
-          : 'cursor:pointer'};
+        color: ${theme[selected ? 'content' : 'contentSecondary'};
```

Comment:
Sorry, the suggestion i made had one missing `]`
```suggestion
        color: ${theme[selected ? 'content' : 'contentSecondary']};
```